docs/add-ons/configuration.md

```diff
@@ -132,7 +132,7 @@ The configuration for an add-on is stored in `config.json`.
 | homeassistant_api | bool | no | This add-on can access to the Home Assistant REST API proxy. Use `http://supervisor/core/api`.
 | docker_api | bool | no | Allow read-only access to Docker API for add-on. Works only for not protected add-ons.
 | privileged | list | no | Privilege for access to hardware/system. Available access: `NET_ADMIN`, `SYS_ADMIN`, `SYS_RAWIO`, `SYS_TIME`, `SYS_NICE`, `SYS_RESOURCE`, `SYS_PTRACE`, `SYS_MODULE` or `DAC_READ_SEARCH`
-| full_access | bool | no | Give full access to hardware like the privileged mode in Docker. Works only for not protected add-ons.
+| full_access | bool | no | Give full access to hardware like the privileged mode in Docker. Works only for not protected add-ons and should not be used! If you using this, you don't need other configs like `uart` or `devices`.
```

Comment:
```suggestion
| full_access | bool | no | Give full access to hardware like the privileged mode in Docker. Works only for not protected add-ons. Consider using other add-on options instead of this, like `devices`. If you enable this option, don't add `devices`, `uart`, `usb` or `gpio` this is not needed.
```